### ISSUE TITLE
I can't find SimpleIdentifiableResourceAssembler

### ISSUE THREAD

**Issue by zachariahyoung** [spring-projects/spring-hateoas-examples#11]
Has this class not been add to the spring hateos project?

**Comment by gregturn** [comment]
No, it hasn't. It's found in this repo's "commons" section.

Essentially, I'm using this example repo as a place to gather feature mods I wish to carry back to Spring HATEOAS and eventually remove from this sample repo.

**Comment by karlnicholas** [comment]
Well, I wouldn't mind being able to setup a demo HAL/HATEOAS project in spring framework but I can't find any documentation on using ResourceAssembler.

**Comment by gregturn** [comment]
`ResourceAssembler` is a Spring HATEOAS class. Documentation can be found here => https://docs.spring.io/spring-hateoas/docs/0.24.0.RELEASE/reference/html/#fundamentals.resource-assembler

**Comment by karlnicholas** [comment]
Hi, thanks Greg. Yes, found it, moved on to ResourceAssemblerSupport class which seems to be doing what I'm trying. Thanks,

**Comment by satishrao84** [comment]
I don't understand. ELI5, where do I find SimpleIdentifiableResourceAssembler? It's not in `org.springframework.hateoas.server` but this repo shows that it *should* be in `org.springframework.hateoas`. I'm very confused.
